streaming/src/main/scala/io/gearpump/streaming/task/ExpressTransport.scala

```diff
@@ -67,63 +60,11 @@ trait ExpressTransport {
 
         val remoteAddress = express.lookupRemoteAddress(transportId)
         if (remoteAddress.isDefined) {
-          if(sendLater.hasPendingMessages){
-            sendLater.flushPendingMessages(remoteAddress.get, transportId)
-          }
           express.transport(taskMessage, remoteAddress.get)
         } else {
-          sendLater.addMessage(transportId, taskMessage)
+          LOG.error(s"Can not find target task $remote")
```

Comment:
Maybe you want to put some inline document about on what ocassion this error will happen?